composer.json

```diff
@@ -91,7 +91,9 @@
 
         "matthimatiker/opcache-bundle": "^0.7",
         "qandidate/toggle-bundle": "^1.1.0",
-        "google/recaptcha": "~1.1"
+        "google/recaptcha": "~1.1",
+        "ggggino/wordbundle": "^1.0",
+        "phpoffice/phpword": "v0.14.*"
```

Comment:
The "v" is not required and should be removed to be consistent with the rest of the file.